Dockerfile

```diff
@@ -20,45 +20,17 @@
 # This is the image we'll use to execute the build (and give it the name 'build').
 # (This image is based on Ubuntu)
 # Fixed version of this in order to have a fixed JDK version
-FROM azul/zulu-openjdk:8 as build
+FROM openjdk:8 as build
 
 # Install some stuff we need to run the build
 RUN apt update -y
 
 # Install general purpose tools
-RUN apt install -y make libpcap-dev libc-dev
-
-# Requied for "with-boost" profile
-RUN apt install -y bison flex gcc g++
-
-# Required for "with-cpp" profile
-RUN apt install -y gcc g++
-
-# Required for "with-proxies" and "with-cpp"
-RUN apt install -y clang
-
-# Required for "with-proxies" and "with-cpp"
-RUN apt install -y cmake
-
-# Required for "with-dotnet" profile
-RUN apt install -y wget
-RUN wget -q https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
-RUN dpkg -i packages-microsoft-prod.deb
-RUN apt install -y software-properties-common
```

Comment:
Could you please undo all of these changes? Most of the installed stuff is needed in order to build the C++, C# and Python stuff, if you remove it you can't build the rest